arrow/arrow.py

```diff
@@ -504,6 +504,7 @@ def span(
         count: int = 1,
         bounds: _BOUNDS = "[)",
         exact: bool = False,
+        weekday: int = 1
```

Comment:
Comma at the end here should fix the lint failure.

Reply:
I added the missing comma. What now?

Reply:
We look over your code :wink: